db/atombios

```diff
@@ -0,0 +1,19 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/radare/radare2-extras"
+R2PM_DESC "[r2-arch] AtomBIOS Disassembler/Analyzer"
+
+R2PM_INSTALL() {
+	./configure --prefix="${R2PM_PREFIX}" || exit 1
+	cd libr || exit 1
+	${MAKE} clean || exit 1
+	${MAKE} atombios R2PM_PLUGDIR="${R2PM_PLUGDIR}" || exit 1
```

Comment:
Will remove this line when I make the atombios-install target depend on the atombios target